Update OpenAI blog credit usage changes (#138)
codedex-io/projects: projects/generate-a-blog-with-openai/generate-a-blog-with-openai.mdx

### OpenAI Account

> Before we do anything, we need an [OpenAI](https://openai.com/api) account. We'll need this account access to an API key that we can use to work with GPT-3. Note that OpenAI no longer offers free credits, so you'll also need to purchase credits to use the API.

Reviewer: ```suggestion
Before we do anything, we need an [OpenAI](https://openai.com/api) account. We'll need this to access an API key for using GPT-3. Note that OpenAI no longer offers free credits, so you'll also need to purchase credits to use the API.
```

Are we able to offer a little more in terms of linking to something that breaks down how pricing/credits works? Or maybe say what the minimum cost would be to go through this tutorial?

Author: The [pricing page](https://openai.com/api/pricing/) is full of tables and formulas that might be a bit much for learners that are just getting started. I'll add a quick note about the $5 minimum to keep it simple.

### Credit Limit

By this point, if you have been playing with the API nonstop, there's a chance that you might have exceeded your purchased credit limit. The following error is thrown when that happens:

```bash
openai.error.RateLimitError:  
You exceeded your current quota, please check your plan and billing details.
```

> If that's the case, go to OpenAI's [Billing overview](https://beta.openai.com/account/billing/overview) page and purchase additional credits.

Reviewer: ```suggestion
If that's the case, go to OpenAI's [Billing overview](https://platform.openai.com/settings/organization/billing/overview) page and purchase additional credits.
```

More current link
